### ISSUE TITLE
Failing to move newly written config from tmp to final location: cross-device link

### ISSUE THREAD

**Issue by bjdag1234** [ncw/rclone#1679]
When filing an issue, please include the following information if possible as well as a description of the problem.  Make sure you test with the latest beta of rclone.

    https://beta.rclone.org/
    https://rclone.org/downloads/

If you've just got a question or aren't sure if you've found a bug then please use the [rclone forum](https://forum.rclone.org/) instead of filing an issue.

> What is your rclone version (eg output from `rclone -V`)

rclone 1.37

> Which OS you are using and how many bits (eg Windows 7, 64 bit)

FreeBSD 10.2 / FreeNAS 10.2

> Which cloud storage system are you using? (eg Google Drive)

Google drive

> The command you were trying to run (eg `rclone copy /tmp remote:tmp`)

```
rclone copy --config api1.conf --fast-list --size-only -v --transfers 64 --checkers 8 --drive-chunk-size=16384k --drive-upload-cutoff=16384k --tpslimit 128 --timeout 1m --stats 30s --log-file /mnt/misc/scripts/restore_script/logs/restore${DATE}.log <path> <path> & disown -a
```

> A log from the command with the `-vv` flag (eg output from `rclone -vv copy /tmp remote:tmp`)

```
Failed to move newly written config from /tmp/api1.conf710941323 to final location: rename /tmp/api1.conf710941323 api1.conf: cross-device link
```

This is on a zfs volume which is different from the root directory. 
We use different API keys per GDrive remote and pair of local-remote paths to avoid API limits on the same google account.

This error stops the transfer due to the config not being written, and also avoids the token being renewed.

The transfer continues if the new config file is written.

**Comment by ncw** [comment]
I think we fixed this already in #1569

Please try [the latest beta](https://beta.rclone.org)

**Comment by bjdag1234** [comment]
Will leave comment in other issue if encountered again.
Closing this in favor of #1569
